class/defaults.yml

```diff
@@ -88,7 +88,7 @@ parameters:
     # Extra volumes that are appended to the helm variable `extraVolumes`
     extraVolumes: {}
     # Extra volume mounts that are appended to the helm variable `extraVolumeMounts`
-    extraVolumeMounts: {}
+    extraVolumeMounts: []
```

Comment:
This looks like a breaking change. Please add a how-to page that instructs how to upgrade to the next major version.

Reply:
@mhutter actually, how does this work with merging from hierarchy? Converting to array would remove the possibility to define volume mounts from commodore defaults yet still add mounts specific to clusters? Or is still possible?

Reply:
Not sure, but what is the use case for hierarchical configuration of volume mounts?

Reply:
One use case would be:
- Define a common theme in the tenant
- Define an additional init container that uses volume mounts for a specific cluster in order to test a Dev build of keycloak extensions

Both themeing and extension make use of volume mounts